utils/constants.tsx

```diff
@@ -523,65 +525,65 @@ export const FileTypes = {
     PATCH: 'patch',
     SVG: 'svg',
     OTHER: 'other',
-};
+} as const;
 
 export const NotificationLevels = {
     DEFAULT: 'default',
     ALL: 'all',
     MENTION: 'mention',
     NONE: 'none',
-};
+} as const;
 
 export const IgnoreChannelMentions = {
     ON: 'on',
     OFF: 'off',
     DEFAULT: 'default',
-};
+} as const;
 
 export const NotificationSections = {
     IGNORE_CHANNEL_MENTIONS: 'ignoreChannelMentions',
     MARK_UNREAD: 'markUnread',
     DESKTOP: 'desktop',
     PUSH: 'push',
     NONE: '',
-};
+} as const;
 
 export const AdvancedSections = {
     CONTROL_SEND: 'advancedCtrlSend',
     FORMATTING: 'formatting',
     JOIN_LEAVE: 'joinLeave',
     PREVIEW_FEATURES: 'advancedPreviewFeatures',
-};
+} as const;
 
 export const RHSStates = {
     MENTION: 'mention',
     SEARCH: 'search',
     FLAG: 'flag',
     PIN: 'pin',
     PLUGIN: 'plugin',
-};
+} as const;
```

Comment:
Allows for `ValueOf<typeof RHSStates>` instead of `'mention' | 'search' | 'flag' | 'pin' | 'plugin'`